test/ir_Airwell_test.cpp

```diff
@@ -143,6 +113,36 @@ TEST(TestDecodeAirwell, SyntheticExample) {
       "m4750s100000",
       irsend.outputStr());
 
+  irsend.reset();
+  irsend.sendAirwell(0x4F2FE7E4);
```

Comment:
Since this is not valid data for airwell, maybe it should be removed, or better replaced with the Nev value for the expected raw.

Reply:
Good point. Done. Value replaced with the new real example added.